test/box-luatest/gh_8817_bug_set_language_shared_test.lua

```diff
@@ -0,0 +1,80 @@
+local t = require('luatest')
+local fio = require('fio')
+local popen = require('popen')
+local server = require('luatest.server')
+
+local g = t.group()
+
+-- Create server test instance.
+g.before_all(function()
+    g.server = server:new({alias = 'test-gh-8817-server'})
+    g.server:start()
+    -- Get port for clients.
+    g.server_port = g.server:exec(function()
+        return require('console').listen(0):name()['port']
+    end)
+end)
+
+-- Stop server test instance.
+g.after_all(function()
+    g.server:stop()
+end)
+
+-- Checks that the language setting is not shared between clients.
+g.test_set_language_not_shared_between_clients = function()
+    -- Find tarantoolctl.
+    -- Probably test-run knows the path to tarantoolctl.
+    local TARANTOOLCTL_PATH = os.getenv('TARANTOOLCTL')
```

Comment:
It is not clear to me why do you use tarantoolctl in the test when the same test could be created without it.
In the nearest time tarantoolctl will be gone and after that we will update all tests that require tarantoolctl. I suspect no one wants to do double work, so let's avoid using tarantoolctl and replace it with a two additional tarantool instances and a couple of connections using `console` module (the same way as in a bug description).

Reply:
Done, now using `require('console').connect` via `test.interactive_tarantool`